src/DSS/DSS.jl

```diff
@@ -142,5 +143,7 @@ for (mv, mv!) in ((:A_ldiv_B,  :A_ldiv_B!),
     end
 end
 
+(\){T}(F::DSSFactor{T}, B::StridedVecOrMat{T}) = A_ldiv_B(F, B)
```

Comment:
Why is `A_ldiv_B` defined? There is a single such method in base, but that is wrong and I'm about to remove it.